packages/core/.storybook/preview.ts

```diff
@@ -2,7 +2,7 @@
 
 import { defineCustomElements } from '../loader';
 import type { Preview, Decorator } from '@storybook/html';
-import { addons } from 'storybook/internal/preview-api';
+import { addons } from 'storybook/preview-api';
```

Comment:
For me this breaks storybook, and the way to fix it is:
```suggestion
import { addons } from 'storybook/internal/preview-api';
```